docs/hol.md

```diff
@@ -0,0 +1,353 @@
+<properties
+   pageTitle="Page title that displays in the browser tab and search results"
```

Comment:
Please put in info here.  You can remove documentationCenter, manager, and editor fields.